checklist.md

```diff
@@ -0,0 +1,50 @@
+- [ ] **Infrastructure as code** is in place (`/infra` folder where applicable, manifest files or code generators in the case of `Aspire` and similar )
+    - [ ] **Bicep Format** - Bicep files should be formatted with `bicep format`. Can possibly be automated with [pre-commit](https://github.com/Azure4DevOps/check-azure-bicep) or [GitHub actions](https://github.com/pamelafox/django-quiz-app/pull/15/files#diff-8af3e80c405f1ab691b04ee13deecae46a34d6e87cc81b9a5f21490bc17e2609R29).
+- [ ] **Workflow** - `.workflow` file is in place (This refers to GitHub Actions `.github/workflows/azure-dev.yml` or custom workflow to run on a GitHub runner)
+- [ ] **Devcontainer** - `.devcontainer` should contain a `devcontainer.json` and `Dockerfile`/`docker-compose.yaml` in order to create a full local dev environment. Start with [azure-dev versions](https://github.com/Azure/azure-dev/tree/cb28058af1e7139be4381532f6b1167d9cd948fb/templates/common/.devcontainer) and modify as needed. See [docker-compose.yaml](https://github.com/Azure-Samples/azure-django-postgres-flexible-appservice/blob/main/.devcontainer/docker-compose_dev.yml) for example that includes a local database service.
+- [ ] **Hook Scripts** - all hook scripts (pre-/post- provisioning and deployment scripts) shall include both sh and pwsh versions.
+- [ ] **Testing** - Minimum coverage tests are in place
+ 
+## Functional requirements
+ 
+- [ ] **azd up** - successfully provisions and deploys a functional app with `azd up`
+- [ ] **azd Pipeline Config** - `.github/workflows` should include [`azure-dev.yaml`](https://github.com/Azure-Samples/todo-python-mongo-aca/blob/main/.github/workflows/azure-dev.yml) to support `azd pipeline config`
+- [ ] **GitHub Actions** run tasks without errors
+- [ ] **DevContainer** has been tested locally and runs
+- [ ] **Codespaces** run [locally and in browser]
+- [ ] **Dashboard** - Resources should include a dashboard so that `azd monitor` works, either by referencing the `monitoring.bicep` module or creating a dashboard separately. See [main.bicep](https://github.com/Azure-Samples/todo-python-mongo-aca/blob/05a7aa59c0253628e293ca0fcc98f35a942df1cc/infra/main.bicep#L129)
+    - [ ] **Monitoring** - Application code should include either OpenCensus or OpenTelemetry so that the monitor is populated. See [todo/app.py](https://github.com/Azure-Samples/todo-python-mongo-aca/blob/05a7aa59c0253628e293ca0fcc98f35a942df1cc/src/api/todo/app.py#L58)
+- [ ] All tests pass
+    
+    In the absense of e2e tests,
+    - [ ] The application has been manually tested to work as per the requirement
+ 
+## Security requirements
+ 
+- [ ] Microsoft Managed Service Identity is implemented
```

Comment:
I would bold "Microsoft Managed Service Identity" or add **Managed Identity** to the beginning to make sure that it stands out since it's an important requirement.

Reply:
Official name is `Managed Identity`